implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -27,6 +27,7 @@ use ockam_api::{
         NodeManagerWorker, NODEMANAGER_ADDR,
     },
 };
+use ockam_api::cli_state::random_name;
```

Comment:
Same here as in the other PR. There is an unused import (`cargo clippy` + `cargo fmt` will help you)